.golangci.yaml

```diff
@@ -0,0 +1,19 @@
+run:
+  timeout: 1m
+  tests: true
+
+linters:
+  enable:
+    - asciicheck
+    - forcetypeassert
+    - gocritic
+    - godot
+    - gofmt
+    - goimports
+    - misspell
+    - stylecheck
+
+linters-settings: # please keep this alphabetized
+  gocritic:
+    disabled-checks:
+      - singleCaseSwitch
```

Comment:
https://github.com/kubernetes-sigs/dra-example-driver/pull/14#discussion_r1194690343
> After poking around - I disabled one check for gocritic - singleCaseSwitch, it is intended to be a single case switch where it complains due to potentially more resource types implementation.

Instead of disabling the check globally, disable it for that particular case switch with `//nolint:gocritic // <some comment explaining why>`.

https://golangci-lint.run/usage/false-positives/#nolint-directive

Reply:
Fixed in https://github.com/kubernetes-sigs/dra-example-driver/pull/18. Removing this config section.